test-eberg.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import logging
+from ArrayTools.matrix import Matrix, Array2D
+
+logging.basicConfig(format='%(asctime)s %(levelname)s %(module)s %(lineno)d: %(message)s', level=logging.DEBUG)
```

Comment:
Logging is an excellent habit to adopt.  It's particularly useful in library functions, which in general, should not print to standard out.  If we set the `level` param, above, to `logging.fatal`, then none of the INFO-level log lines will print, or you can crank up the log level to DEBUG and see everything, which is good for debugging. As you can imagine, i have a lot of opinions on logging, both as a developer and as someone who supports apps written by others.